test/smoke/testApps/SimpleEventData/build.gradle

```diff
@@ -0,0 +1,25 @@
+// apply plugin: 'java-library'
+apply plugin: 'war'
+
+dependencies {
+    compile project(':core')
```

Comment:
in the test applications, we need to use the variables created in _testApps/build.gradle_. Instead of `compile project(':core')` use `compile aiCoreJar`. Instead of `compile project(':web')` use `compile aiWebJar`

Reply:
Thanks, I will change it in code.